amida-tech/indaba-client
# Feature/INBA-675 Project activation errors

#### What does this PR do?
Passes readable project activation error messages from greyscale up to a toast. Displays a generic error message when the error from greyscale is not recognized.

#### Related JIRA tickets:
[INBA-675](https://jira.amida-tech.com/browse/INBA-675)

#### How should this be manually tested?
1. Create a project in the wizard, set the name but do not add questions, and do not add anything else.
1. Open the project in the project manager.
1. Try to activate the project
1. Check that you get a meaningful error message.
1. Resolve the error
1. Repeat the previous two steps until the project successfully activates. 

#### Background/Context
1. This displays text from the backend error messages directly to the user, bypassing i18n. In the future, we should return an integer error code for each specific error case from greyscale, which the client can use to index the vocabulary instead. This is not ticketed.
1. A project without any survey returns a 500 when the greyscale request to the survey service fails. We should change greyscale catch the error and return a 4xx error in this case. This is also unticketed.

#### Screenshots (if appropriate):
